test/regress/regress0/expect/scrub.07.sy

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --cegqi-si=all --no-dump-synth
+; COMMAND-LINE: --cegqi-si=all --sygus-out=status
```

Comment:
Could we change some of the regressions to use the different output modes or is it difficult to match the expected output with the actual output?

Reply:
Yes, the actual output can fluctuate, which is why I haven't enabled printing of synthesis solutions.  This is the same reason we don't typically match the output of get-model. 
This is indeed a major shortcoming of the regressions though.  One way to address this is to have a --check-synth-fun analogous to --check-model.  This is also an open issue:  https://github.com/CVC4/CVC4/issues/1143